lisa/core/environment.py

```diff
@@ -102,23 +102,25 @@ def getNodeByName(self, name: str, throwError: bool = True) -> Optional[Node]:
         found = None
 
         if len(self.nodes) == 0:
-            raise Exception("nodes shouldn't be Empty when call getNodeByName")
-        else:
-            for node in self.nodes:
-                if node.name == name:
-                    found = node
-                    break
-            if throwError:
-                raise Exception(f"cannot find node {name}")
+            raise LisaException("nodes shouldn't be Empty when call getNodeByName")
+
+        for node in self.nodes:
+            if node.name == name:
+                found = node
+                break
+        if found is None and throwError:
```

Comment:
Sometimes, the upper level method need to suppress exception. Some situation is an exception in most time, but may be normal in some cases. In this case, hard code name in an environment is not a good practice, since it means the case is hard to migrate to other environments. But it's not figured out how to find a node match some criterias yet, and I don't want this part is complex. It may be changed in future.